android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -687,6 +687,11 @@ public void startActivityForPluginWithResult(PluginCall call, Intent intent, int
      */
 
     public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+        if (requestCode == BridgeWebChromeClient.GEOLOCATION_REQUEST_PERMISSIONS) {
```

Comment:
the code should be unique, if you use the same on the geolocation plugin then only one of them will receive the response

Reply:
Yeah. Will make sure it is different